app/config.pri.default

```diff
@@ -1,5 +1,8 @@
 BASE_DIR=</home/folder>/Projects/quick
 
+# Default Mergin URL - used with /auth/signup
+DEFAULT_MERGIN_URL=<MERGIN URL>
```

Comment:
Let's hardcode the default mergin URL in MerginApi to https://public.cloudmergin.com/

It could be e.g. `static QString defaultApiRoot() { return "...."; }` in MerginApi